From ea68a34fdfd708782d1868a816f0bd166103c504 Mon Sep 17 00:00:00 2001
From: Jonathan David <jonathan.david@ni.com>
Date: Fri, 18 Jul 2014 15:28:22 -0500
Subject: [PATCH 1/4] Remove ability to close keyboard

Do not allow the user to exit Florence. If the user closes Florence
and has no access to a physical keyboard, it may be possible that
they cannot create a new instance of Florence. It is expected that
the user would either have no need for a virtual keyboard or would
know how to access Florence if a physical keyboard is attached.

Signed-off-by: Jonathan David <jonathan.david@ni.com>
Signed-off-by: Ioan-Adrian Ratiu <adrian.ratiu@ni.com>
---
 data/layouts/compact-alt.xml.in.in  | 10 +++++-----
 data/layouts/compact.xml.in.in      | 10 +++++-----
 data/layouts/florence-alt.xml.in.in | 10 +++++-----
 data/layouts/florence.xml.in.in     | 12 ++++++------
 data/layouts/mini.xml.in.in         |  4 ----
 5 files changed, 21 insertions(+), 25 deletions(-)

diff --git a/data/layouts/compact-alt.xml.in.in b/data/layouts/compact-alt.xml.in.in
index a9d3f39..96b77a4 100644
--- a/data/layouts/compact-alt.xml.in.in
+++ b/data/layouts/compact-alt.xml.in.in
@@ -669,19 +669,19 @@
 	<height>10</height>
 	<key>
 		<action>reduce</action>
-		<modifier>
-			<code>8</code> <!-- alt -->
+		<!-- <modifier>
+			<code>8</code>
 			<action>close</action>
-		</modifier>
+		</modifier> -->
 		<xpos>1</xpos>
 		<ypos>1</ypos>
 	</key>
 	<key>
 		<action>config</action>
-		<modifier>
+		<!-- <modifier>
 			<code>8</code>
 			<action>switch</action>
-		</modifier>
+		</modifier> -->
 		<xpos>1</xpos>
 		<ypos>3</ypos>
 	</key>
diff --git a/data/layouts/compact.xml.in.in b/data/layouts/compact.xml.in.in
index 949e09b..330e995 100644
--- a/data/layouts/compact.xml.in.in
+++ b/data/layouts/compact.xml.in.in
@@ -669,19 +669,19 @@
 	<height>10</height>
 	<key>
 		<action>reduce</action>
-		<modifier>
-			<code>8</code> <!-- alt -->
+		<!-- <modifier>
+			<code>8</code>
 			<action>close</action>
-		</modifier>
+		<</modifier> -->
 		<xpos>1</xpos>
 		<ypos>1</ypos>
 	</key>
 	<key>
 		<action>config</action>
-		<modifier>
+		<!-- <modifier>
 			<code>8</code>
 			<action>switch</action>
-		</modifier>
+		</modifier> -->
 		<xpos>1</xpos>
 		<ypos>3</ypos>
 	</key>
diff --git a/data/layouts/florence-alt.xml.in.in b/data/layouts/florence-alt.xml.in.in
index 715e120..577194b 100644
--- a/data/layouts/florence-alt.xml.in.in
+++ b/data/layouts/florence-alt.xml.in.in
@@ -626,19 +626,19 @@
 	<height>10</height>
 	<key>
 		<action>reduce</action>
-		<modifier>
-			<code>8</code> <!-- alt -->
+		<!-- <modifier>
+			<code>8</code>
 			<action>close</action>
-		</modifier>
+		</modifier> -->
 		<xpos>1</xpos>
 		<ypos>1</ypos>
 	</key>
 	<key>
 		<action>config</action>
-		<modifier>
+		<!-- <modifier>
 			<code>8</code>
 			<action>switch</action>
-		</modifier>
+		</modifier> -->
 		<xpos>1</xpos>
 		<ypos>3</ypos>
 	</key>
diff --git a/data/layouts/florence.xml.in.in b/data/layouts/florence.xml.in.in
index 32b6eb9..e935a88 100644
--- a/data/layouts/florence.xml.in.in
+++ b/data/layouts/florence.xml.in.in
@@ -626,19 +626,19 @@
 	<height>10</height>
 	<key>
 		<action>reduce</action>
-		<modifier>
-			<code>8</code> <!-- alt -->
+		<!-- <modifier>
+			<code>8</code>
 			<action>close</action>
-		</modifier>
+		</modifier> -->
 		<xpos>1</xpos>
 		<ypos>1</ypos>
 	</key>
 	<key>
 		<action>config</action>
-		<modifier>
-			<code>8</code> <!-- alt -->
+		<!-- <modifier>
+			<code>8</code>
 			<action>switch</action>
-		</modifier>
+		</modifier> -->
 		<xpos>1</xpos>
 		<ypos>3</ypos>
 	</key>
diff --git a/data/layouts/mini.xml.in.in b/data/layouts/mini.xml.in.in
index f10fcee..24b74cb 100644
--- a/data/layouts/mini.xml.in.in
+++ b/data/layouts/mini.xml.in.in
@@ -34,10 +34,6 @@
 	<height>3</height>
 	<key>
 		<action>reduce</action>
-		<modifier>
-			<code>8</code> <!-- alt -->
-			<action>close</action>
-		</modifier>
 		<xpos>1</xpos>
 		<ypos>1</ypos>
 	</key>
-- 
2.13.2

